test/Microsoft.VisualStudio.Validation.Tests/AssertDialogSuppression.cs

```diff
@@ -17,20 +17,24 @@ internal class AssertDialogSuppression : IDisposable
     /// <summary>
     /// Stores the original popup-ability of the assertion dialog.
     /// </summary>
-    private bool? originalAssertUiSetting;
+    private bool? originalAssertUiSetting = false;
```

Comment:
nit: Why not simply making this not nullable? It would default to false and we could delete condition in line 46. Unless I'm missing something.